app/views/tag/_tagging.html.erb

```diff
@@ -72,4 +79,4 @@ $(".label").on("click", function(e){
 </form>
 <% end %>
 
-<%= render partial: 'tag/location' %>
```

Comment:
Restore this to the previous version as this is unrelated to your changes.

Reply:
I think it's a difference in the line number, I am able to see this in the next line, I haven't removed it altogether, its just in the next line I think.